tests/main.test.ts

```diff
@@ -111,6 +111,7 @@ describe("User tests", () => {
     const result = await workerFetch.fetch(
       {
         method: "GET",
+        url: "http://localhost:4000",
```

Comment:
What's this for?

Reply:
This is a dummy url as it expected that an url is provided within a `Request` object. Before we did not serve a manifest on `GET` but now we do so that value is required for the test to pass.